src/__tests__/Search/CompareWithBase.test.tsx

```diff
@@ -168,7 +175,7 @@ describe('Compare With Base', () => {
     ).not.toBeInTheDocument();
 
     const frameworkDropdown = screen.getAllByRole('button', {
-      name: 'Framework talos',
+      name: 'Framework Framework',
```

Comment:
isn't this a bit curious?

Reply:
Yes, it has something to do with the labels for the Search page vs Results page-- I'll look over for next PR re styles coming up.